### PR TITLE
Tsug 06

### PR DESCRIPTION
**Pull request by pjazdzewski1990**

### CODE REVIEW

File: _posts/2015-07-11-tsug_06.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+First was Nuno Teixeira ([you can find him here](https://twitter.com/nmat)) from [Codacy](https://www.codacy.com/), who talked about their Scala based stack and how they approached scaling and growing their application. He showed a bit of their codebase, talked about the problems they had, how they manage to fix them and about their plans for the future. The presentation is available [here](https://goo.gl/q5ODFJ)
+
+After that our Scala hAkker Patryk Jażdżewski, gave a talk "Scala on Android" in which he explained how we can create applications on Android using [Scaloid](https://github.com/pocorall/scaloid). In this whirlwind tour Patryk's walks through recognized Scala features and shows how Scaloid is using them to make Android programming more user friendly. You can check his slides [here](https://docs.google.com/presentation/d/1PeuEs_phiSH1MJ8y9MtEQwpL_OJuvkk3eiNvGJxlsEM/edit?usp=sharing)
```
Comment (LukasGasior1) [inline _posts/2015-07-11-tsug_06.md]: Patryk's -> Patryk

File: _posts/2015-07-11-tsug_06.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+tags:
+- Scala
+- TSUG
+- Android
+- Scaloid
+---
+
+## <a name="summary" href="#summary">Our meeting</a>
+
+Tricity Scala User Group is an initiative started by scalac to promote Scala, it's ecosystem and functional programming in general. Last week  we held another TSUG meeting. This time we met in Dwie Zmiany club in Sopot. Although it was Friday we had a room full of Scala enthusiasts eager to learn and share their knowledge. We had two speakers to give interesting talks. After that we spend time on networking, drinking free beer and eating scalac-cookies :)
```
Comment (LukasGasior1) [inline _posts/2015-07-11-tsug_06.md]: it's ecosystem -> its ecosystem

Comment (mareksuscak) [inline _posts/2015-07-11-tsug_06.md]: So are we using `Scalac` or `scalac`? I think we should go with the capital lettered version.

"Last week  we held..." - two spaces between "week" and "we"

"After that we spend..." -> use past tense and i would reword it to "After that we spent the time networking, drinking free beer and eating scalac-cookies" -> you can eliminate the preposition entirely and I like this form much more

File: _posts/2015-07-11-tsug_06.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+First was Nuno Teixeira ([you can find him here](https://twitter.com/nmat)) from [Codacy](https://www.codacy.com/), who talked about their Scala based stack and how they approached scaling and growing their application. He showed a bit of their codebase, talked about the problems they had, how they manage to fix them and about their plans for the future. The presentation is available [here](https://goo.gl/q5ODFJ)
```
Comment (mareksuscak) [inline _posts/2015-07-11-tsug_06.md]: "...how they manage..." -> "...how they managed..." ?

File: _posts/2015-07-11-tsug_06.md (hunk 1 above)
Comment (mareksuscak) [inline _posts/2015-07-11-tsug_06.md]: "Patryk's" -> "Patryk"
"walks through" -> "walks through the"
"user friendly" -> "user-friendly" -> http://www.thefreedictionary.com/user-friendly

File: _posts/2015-07-11-tsug_06.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+After that our Scala hAkker Patryk Jażdżewski, gave a talk "Scala on Android" in which he explained how we can create applications on Android using [Scaloid](https://github.com/pocorall/scaloid). In this whirlwind tour Patryk's walks through recognized Scala features and shows how Scaloid is using them to make Android programming more user friendly. You can check his slides [here](https://docs.google.com/presentation/d/1PeuEs_phiSH1MJ8y9MtEQwpL_OJuvkk3eiNvGJxlsEM/edit?usp=sharing)
+
+We hope to see you at the next meeting! You can join us on [Metup](http://www.meetup.com/Tricity-Scala-Users-Group/)
```
Comment (mareksuscak) [inline _posts/2015-07-11-tsug_06.md]: "Metup" -> "Meetup"

**Comment by pjazdzewski1990** [comment]
@LukasGasior1 @mareksuscak Thanks
